chapters/sampling-randomization-power.tex

```diff
@@ -1,391 +1,434 @@
+  \index{sampling}
+That master list may be called a \textbf{sampling universe}, a \textbf{listing frame}, or something similar.
+We recommend that this list be organized in a \textbf{master data set}\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Master_Data_Set}},
+creating an authoritative source for the existence and fixed 
+characteristics of each of the units that may be surveyed.\sidenote{
+	\url{https://dimewiki.worldbank.org/wiki/Unit_of_Observation}}
+The master data set indicates how many individuals are eligible for data collection,
+and therefore contains statistical information about the likelihood that each will be chosen.
+
+The simplest form of random sampling is \textbf{uniform-probability random sampling}.
+This means that every observation in the master data set
+has an equal probability of being included in the sample.
+The most explicit method of implementing this process
+is to assign random numbers to all your potential observations,
+order them by the number they are assigned,
+and mark as `sampled' those with the lowest numbers, up to the desired proportion.
+(In general, we will talk about sampling proportions rather than numbers of observations.
+Sampling specific numbers of observations is complicated and should be avoided,
+because it will make the probability of selection very hard to calculate.)
+There are a number of shortcuts to doing this process,
+but they all use this method as the starting point,
+so you should become familiar with exactly how this method works.
+
+Almost all of the relevant considerations for sampling come from two sources:
+deciding what population, if any, a sample is meant to represent (including subgroups);
+and deciding that different individuals should have different probabilities
+of being included in the sample.
+These should be determined in advance by the study design,
+since otherwise the sampling process will not be clear,
+and the interpretation of measurements is directly linked to who is included in them.
+Often, data collection can be designed to keep complications to a minimum,
+so long as it are carefully thought through from this perspective.
+Ex post changes to the study scope using a sample drawn for a different purpose
+usually involve tedious calculations of probabilities and should be avoided.
+
+\subsection{Randomization}
+
+\textbf{Randomization}, in this context, is the process of assigning units into treatment arms.
+Most of the Stata commands used for sampling can be directly transferred to randomization,
+since randomization is also a process of splitting a sample into groups.
+Where sampling determines whether a particular individual
+will be observed at all in the course of data collection,
+randomization determines what state each individual will be observed in.
+Randomizing a treatment guarantees that, \textit{on average},
 the treatment will not be correlated with anything it did not cause.\cite{duflo2007using}
```

Comment:
_anything it did not cause_ is another example of where we can be more explicit to be more accessible to someone new to the topic. 

While not a linguistically beautiful, I think this has less risk of confusion:

_the treatment will not be correlated with anything but the result of that treatment_

Reply:
👍🏼 631fa1ea504c137bb2739425d21e100e14f07365